pkg/migrations/sql/idempotent/base.sql

```diff
@@ -2507,17 +2600,30 @@ BEGIN
         SELECT *
         FROM unnest(remaining_metrics)
     LOOP
+        IF log_verbose THEN
+            lockStartT := clock_timestamp();
+            RAISE LOG 'promscale maintenance: compression: metric %: waiting for lock', r.metric_name;
+        END IF;
```

Comment:
A double-check: are we fine with too much logging here, since it's a loop and people will have lakhs of metrics. I understand it's verbose only, but a double-check with our motive here.

Reply:
yes that's the point here